### DISCUSSION TITLE
Full Width Indicator without Labels

### DISCUSSION THREAD

**Discussion by AaronMaria** [ramensoftware/windhawk-mods#404] [Q&A] [answered]
With 1.2.1 it seems no longer possible to use the "Taskbar Labels for Windows 11"-Mod without actually showing Labels. Am I missing something?
Alternatively is it possible to get the "Full Width Indicator"-Functionality with the "Windows 11 Taskbar Styler"-Mod?

**Comment by m417z** [comment]
This was never officially supported. After all, the mod is called "Taskbar Labels for Windows 11". I'll see what I can do.

**Comment by m417z** [comment, marked as answer]
As a temporary fix, you can do the following:
* Disable the mod.
* Fork the mod.
* Comment or remove these lines:
https://github.com/ramensoftware/windhawk-mods/blob/b9c4fefe6754408557a92715d6f4e11465f8ae3a/mods/taskbar-labels.wh.cpp#L1200-L1205
* Set width to `0` in mod settings.
* Set grouping mode to "Always" in taskbar properties.

**Comment by AaronMaria** [comment]
Thank you, for the quick help, it is working as expected.

**Comment by m417z** [comment]
In version 1.2.2 of the mod, it's now possible to have no labels by setting the width to -1 in the mod settings.
